components/uikit/button/index.tsx

```diff
@@ -1,11 +1,11 @@
-type Props = React.ButtonHTMLAttributes<HTMLButtonElement>;
+type Props = React.ButtonHTMLAttributes<HTMLButtonElement>
 
 export const Button = (props: Props) => {
   return (
     <button
       {...props}
       type={props.type ?? "button"}
-      className={`bg-sky-500 hover:bg-sky-700 text-white font-bold py-2 px-4 rounded ${props.className}`}
+      className={`bg-sky-400 hover:bg-sky-500 text-white font-bold py-2 px-4 rounded ${props.className}`}
     />
-  );
-};
+  )
+}
```

Comment:
You should use destructuring for props:

`const { type = "button", className } = props`